agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -381,7 +371,7 @@ impl MpidMessage {
                                                      public_key) && self.header.verify(public_key)
     }
 
-    fn encode(recipient: &::NameType, body: &Vec<u8>) -> Vec<u8> {
+    fn encode(recipient: &XorName, body: &Vec<u8>) -> Vec<u8> {
         ::utils::encode(&(recipient, body)).unwrap_or(vec![])
```

Comment:
serialise/deserialise are in maidsafe_utils and this will work here (traits should all be satisfied).